.travis.yml

```diff
@@ -2,9 +2,9 @@ os: linux
 dist: xenial
 language: go
 go:
-  - 1.11.x
-  - 1.12.x
   - 1.13.x
+  - 1.14.x
+  - 1.15.x
```

Comment:
No need to modify the CI in this PR. Please open another PR to make those changes.

Reply:
Yeah I'll take that out. Was just trying to figure out why the tests were passing for me on my machine. Was worried it was a Windows 10 issue